sdcm/cluster.py

```diff
@@ -398,13 +398,13 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
         self.parent_cluster = parent_cluster  # reference to the Cluster object that the node belongs to
         self.logdir = os.path.join(base_logdir, self.name)
         makedirs(self.logdir)
+        self.log = SDCMAdapter(LOGGER, extra={'prefix': str(self)})
```

Comment:
Why do you move this line?

Reply:
I think, we should initiate self.log on earlier, once main properties are defined

Reply:
ok